src/storage/query/ScanEdgeProcessor.cpp

```diff
@@ -179,6 +179,11 @@ void ScanEdgeProcessor::runInMultipleThread(const cpp2::ScanEdgeRequest& req) {
   folly::collectAll(futures).via(executor_).thenTry([this](auto&& t) mutable {
     CHECK(!t.hasException());
     const auto& tries = t.value();
+    size_t sum = 0;
+    for (size_t j = 0; j < tries.size(); j++) {
+      sum += results_[j].size();
+    }
+    resultDataSet_.rows.reserve(sum);
     for (size_t j = 0; j < tries.size(); j++) {
       CHECK(!tries[j].hasException());
```

Comment:
Why do your remove `CHECK(!tries[j].hasException());` in L133 in GetPropProcessor.cpp while not here?

Reply:
👌 , will keep consistent~